### PR TITLE
[GEOS-9204] Complex GeoJSON reports a @dataType for complex types with simple contents, when the content is missing

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

File: src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/GeoJsonOutputFormatWfsTest.java [hunk 1]
```diff
@@ -91,8 +91,19 @@ public void testGetGeoJsonResponseWfs20() throws Exception {
                 getAsJSON(
                         "wfs?request=GetFeature&version=2.0"
                                 + "&typenames=st_gml32:Station_gml32&outputFormat=application/json");
+        print(response);
```
Comment (nmco) [inline src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/GeoJsonOutputFormatWfsTest.java]: A leftover?

**Comment by nmco** [comment]
Looks good, thx Andrea :smiley:

File: src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/GeoJsonOutputFormatWfsTest.java (hunk 1 above)
Comment (aaime) [reply src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/GeoJsonOutputFormatWfsTest.java]: Yes, a leftover indeed. I would not want you to get bored :-p

**Comment by aaime** [comment]
Leftover removed, merging

**Pull request merged**